### PR TITLE
change view button theme and style.

### PR DESCRIPTION
**Pull request by Himanshunegi0332**
Targets #1694 

### Description
change_view button theme and style in HR analytics section.

### Checklist:
 I have performed a self-review of my own code.

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<IfModule mod_rewrite.c>
```
Comment (rathorevaibhav) [inline .htaccess:1]: what does this file do?

File: resources/views/hr/recruitment/reports.blade.php [hunk 2]
```diff
@@ -4,7 +4,8 @@
 
 <div class="chart-container" align="center">
     <div class="bar-chart-container">
-        <h2 align="left">Analytics</h2>
+        <div ="button" align="left">
```
Comment (rathorevaibhav) [inline resources/views/hr/recruitment/reports.blade.php:7]: Few questions:
1. I think you missed closing tag `</div>`
2. What does `="button"` do?

File: resources/views/hr/recruitment/reports.blade.php [hunk 3]
```diff
@@ -4,7 +4,8 @@
...
+        <h2 href="button" class="btn btn-primary text- white">Analytics</h2>
```
Comment (rathorevaibhav) [inline resources/views/hr/recruitment/reports.blade.php:8]: Few points & questions:
1. What does `href` attribute does in `<h2>`? can you please share your understanding?
2. Does `text- white` class work? With a space in between?
3. If this element is supposed to be a button, I think you should use `<button>` HTML tag

**Review by rathorevaibhav — CHANGES_REQUESTED** [review]

File: .htaccess [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+RewriteEngine On
+# RewriteBase /
+# NOT needed unless you're using mod_alias to redirect
+
+RewriteCond %{REQUEST_URI} !/public
+RewriteRule ^(.*)$ public/$1 [L]
+# Direct all requests to /public folder
+
+</IfModule>
```
Comment (P4NK4J) [inline .htaccess:13]: What's the use of this file ?

File: resources/views/hr/recruitment/reports.blade.php (hunk 3 above)
Comment (P4NK4J) [inline resources/views/hr/recruitment/reports.blade.php:8]:
```suggestion
        <h2 href="button" class="btn btn-primary text-white">Analytics</h2>
```

**Review by P4NK4J — CHANGES_REQUESTED** [review]
Left some comments, please check

**Pull request closed without merging**